.github/workflows/pypi.yml

```
@@ -0,0 +1,34 @@
+name: Publish to PyPI
+
+on:
+  push:
+    branches:
+      - 'master'
+    tags: [ 'v*' ]
+
+env:
+  DEFAULT_PYTHON: 3.9
+
+jobs:
+  publish:
+    name: Publish to PyPI
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v2
+    - name: Set up Python
+      uses: actions/setup-python@v2
+      with:
+        python-version: ${{ env.DEFAULT_PYTHON }}
+    - name: Install dependencies
+      run: |
+        pip install --upgrade pip build setuptools wheel
+        pip install -r requirements-dev.txt
```

You only need build, no need to install dev deps, only build deps are necessary. If there's some build-dep you need, add it to `pyproject.toml` instead.
```suggestion
        pip install --upgrade build
```